src/style.css

```diff
@@ -1,3 +1,78 @@
+:root {
+  --active-color: #00acdc;
+  --header-height: 60px;
+  --logo-size: 40px;
+  --header-font-size: 12px;
+  --corner-radius: 8px;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  font-family: Roboto, serif;
```

Comment:
The font-family is set to 'Roboto, serif'. If 'Roboto' is not available, the fallback 'serif' might not match the intended design. Consider using a more suitable fallback font, such as 'sans-serif'.